openhtf/util/validators.py

```diff
@@ -62,6 +62,14 @@ def MyPhase(test):
 
 _VALIDATORS = {}
 
+class ValidatorException()
```

Comment:
is this valid syntax?  when i run it on its own it doesn't execute.  Also, should you inherit from the Exception base class?

Reply:
Yep, should be inheriting from Exception. I clearly remember thinking that I needed to remember to do that and obviously I did not remember hard enough.